contracts/optimistic-ethereum/OVM/bridge/OVM_BaseCrossDomainMessenger.sol

```diff
@@ -40,7 +40,7 @@ abstract contract OVM_BaseCrossDomainMessenger is iOVM_BaseCrossDomainMessenger,
         address _target,
         bytes memory _message,
         uint32 _gasLimit
-    ) nonReentrant
+    )
```

Comment:
OK, I've removed `nonReentrant` from `sendMessage` and `replayMessage`. This _feels_ safe because on L1 those functions both call only `OVM_CanonicalTransactionChain`, and on L2 we only have `sendMessage` which calls to the `OVM_L2ToL1MessagePasser`.